lib/quicly.c

```diff
@@ -1994,6 +2068,9 @@ static int do_detect_loss(quicly_loss_t *ld, int64_t now, uint64_t largest_acked
         quicly_acks_release(&conn->egress.acks, &iter);
         quicly_acks_next(&iter);
     }
+    conn->egress.cc.bytes_in_flight -= conn->egress.cc.this_ack.nbytes;
+    if (conn->egress.cc.this_ack.nbytes != 0 && conn->egress.loss.rto_count == 0)
+        cc_cong_signal(&conn->egress.cc.ccv, CC_FIRST_RTO, (uint32_t)conn->egress.cc.bytes_in_flight);
```

Comment:
What does this function do?

Reply:
My understanding is that it signals the congestion controller that we have observed first RTO.